randovania/data/json_data/prime3.txt

```diff
@@ -2590,14 +3112,23 @@ Asset id: 5767114369935179109
 > Dock to Zipline Station Alpha; Heals? False
   * Normal Door to Zipline Station Alpha/Dock to Main Docking Access
   > Dock to Main Docking Bay
-      Grapple Swing or Use Screw Attack (Space Jump)
+      All of the following:
+          After Main Docking Access Phazon Vines
+          Grapple Swing or Use Screw Attack (Space Jump)
 
 > Dock to Main Docking Bay; Heals? False
   * Normal Door to Main Docking Bay/Dock to Main Docking Access
   > Dock to Zipline Station Alpha
       All of the following:
-          HyperModeTank and Hyper Ball
+          After Main Docking Access Phazon Vines
           Grapple Swing or Use Screw Attack (Space Jump)
+  > Event - Phazon Vines
+      HyperModeTank and Hyper Ball
+
+> Event - Phazon Vines; Heals? False
+  * Event Main Docking Access Phazon Vines
+  > Dock to Main Docking Bay
+      Trivial
 
 ----------------
 Security Station
```

Comment:
This room is missing requirements

Reply:
All the docks should be connected now? Not sure if "Spire Side" is even really necessary since you can get anywhere to anywhere (except up to security tube)

Reply:
Path to ship missile from landing site a need melt objects requirement

Reply:
2nd half off the above comment needs to be resolved still